style.md

```diff
@@ -0,0 +1,672 @@
+}
+
+// This is a helper function used by myVeryComplicatedProcedure.
+// [Comment explaining the second task]
+function doTheSecondTask(): void {
+    // Imagine this function body is 100 lines long!
+    doOtherStuff();
+}
+
+// This is a helper function used by myVeryComplicatedProcedure.
+// [Comment explaining the third task]
+function doTheThirdTask(): void {
+    // Imagine this function body is 100 lines long!
+    doYetMoreStuff();
+}
+
+// [Comment explaining the purpose of my very complicated function]
+// Imagine that this function body would be a dense, unreadable 300 lines
+// if it wasn't broken up into several helper function calls this way!
+function myVeryComplicatedProcedure(): void {
+    // [Comment briefly summarizing the first task]
+    doTheFirstTask();
+    // [Comment briefly summarizing the second task]
+    doTheSecondTask();
+    // [Comment briefly summarizing the third task]
+    doTheThirdTask();
+}
+```
+
+## Pad infix operators with spaces
+
+An infix operator should have whitespace separating it from both operands.
+
+``` ts
+const mySum: number = 100 + 200;
+```
+
+``` ts
+const myValue: number = myNumberArray[myIndex + 10];
+```
+
+## Use parentheses generously when mixing infix operators
+
+When a single expression contains several different infix operators, each group of identical operators should generally be enclosed within parentheses `()`. This helps to avoid any confusion or ambiguity regarding the intended order of operations.
+
+``` ts
+const myValue: number = 10 * (20 + myOtherValue);
+```
+
+``` ts
+const myBoolean: boolean = (
+    (firstCondition || secondCondition) &&
+    (thirdCondition || fourthCondition)
+);
+```
+
+## Use space after colons but not before
+
+In objects and in typed declarations, there should be a single space after each colon `:` and no space before it.
+
+``` ts
+const myObject: Object {
+    myAttribute: "hello world",
+};
+```
+
+## Prefer double-quoted strings
+
+String literals should be double-quoted `""`. In general, they should not ever be single-quoted `''`.
+Template strings (enclosed within backticks ``` `` ```) should generally not be used for string literals that do not actually contain any interpolation.
+
+``` ts
+const myString: string = "You say \"goodbye\" and I say \"hello\".";
+```
+
+## Use concatenation when writing long string literals
+
+String literals that are too long to fit on a single line should be spread across multiple lines by concatenating many shorter string literals.
+
+``` ts
+const myLongStringLiteral = (
+    "This is a long string literal. In fact, it is so long, that it could " +
+    "not possibly fit on a single line and still be readable. " +
+    `Remember that the recommended maximum line length is ${MaxLineLength} ` +
+    "characters!"
+);
+```
+
+## Prefer braces around arrow function bodies
+
+In general, the bodies of arrow functions should always be enclosed within
+braces `{}`.
+
+``` ts
+myArray.map(value => {
```

Comment:
Although parentheses around a single-parameter arrow function are not required, I usually do it anyway for consistency.

Reply:
I can see the logic for that. I will add it as a guideline.